### PR TITLE
[cleanup] Moved kodi properties parser to utils

### PR DESCRIPTION
**Pull request by CastagnaIT**
Cleanup to kodi properties parser and relative little cleanup to related code/contructors
to removes all incomprehensible variable names and make everything more uniform
to better understand where the data comes from,
the second thing is allow to get/propagate the properties easily with less variables

### CODE REVIEW

File: src/test/TestHelper.h [hunk 1]
```diff
@@ -75,10 +76,13 @@ class AESDecrypter : public IAESDecrypter
...
-  uint64_t GetNowTime() override { return mock_time; }
-  std::chrono::system_clock::time_point GetNowTimeChrono() { return mock_time_chrono; };
+  DASHTestTree(UTILS::PROPERTIES::KodiProperties kodiProps);
+  uint64_t GetNowTime() override { return m_mock_time; }
+  void SetNowTime(uint64_t time) { m_mock_time = time; }
   void SetLastUpdated(std::chrono::system_clock::time_point tm) override { lastUpdated_ = tm; };
+  std::chrono::system_clock::time_point GetNowTimeChrono() { return m_mock_time_chrono; };
+
+private:
+  uint64_t m_mock_time = 10000000L;
```
Comment (glennguy) [inline src/test/TestHelper.h]: no big deal but should we be dropping the underscores in the name i.e `m_mockTime` ?

Comment (CastagnaIT) [reply src/test/TestHelper.h]: yes my intention but also my oversight :-D

Comment (glennguy) [reply src/test/TestHelper.h]: All good! I'm not too worried about getting everything perfect first time. We can always do a couple of cleanups later on to get things that were missed beforehand.

**Review by glennguy — APPROVED** [review]
Looks great!

**Pull request merged**